Formula/k/kubecolor.rb

```diff
@@ -0,0 +1,26 @@
+class Kubecolor < Formula
+  desc "Colorize your kubectl output"
+  homepage "https://kubecolor.github.io/"
+  url "https://github.com/kubecolor/kubecolor/archive/refs/tags/v0.2.2.tar.gz"
+  sha256 "ba0894a8e26fefff47a0691529964303bdd8fdc2d7ce74e7d241cb5a2f2ade50"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    ldflags = %W[
+      -s
+      -w
+      -X main.Version=v#{version}
+    ]
+
+    system "go", "build", *std_go_args(output: bin/"kubecolor", ldflags:)
+  end
+
+  test do
+    assert_match "v#{version}", shell_output("#{bin}/kubecolor --kubecolor-version 2>&1")
```

Comment:
need a better test?

Reply:
as explained, there's nothing to test as it just re-write the output of `kubectl` with colors... at least we test that the binary is working...

Reply:
Please test that it does. Version tests rarely show if a formula is really working

Reply:
@SMillerDev, `version` is the only thing the binary can output, or I will need to install a local kubernetes cluster + resources + `kubectl` then I should be able to run a `kubectl` command that will output something with color that I can test... 

So no, it's not possible to test anything else... executing the binary to display the build version is all we can do...

Reply:
One thing we could do to test the integration is something like:

```bash
output=$(KUBECTL_COMMAND='echo' kubecolor get pods -o yaml)

if [[ $output == 'get pods -o yaml' ]]; then
  echo "works"
fi
```

Sorry above example is in Bash. I don't know how Ruby or Homebrew tests work. But maybe something like this?

```suggestion
    assert_match "v#{version}", shell_output("#{bin}/kubecolor --kubecolor-version 2>&1")
    # kubecolor should consume the '--plain' flag
    assert_match "get pods -o yaml", shell_output("KUBECTL_COMMAND=echo #{bin}/kubecolor get pods --plain -o yaml")
```